src/SolutionAssemblyInfo.cs

```diff
@@ -1,38 +0,0 @@
-﻿// --------------------------------------------------------------------------------------------------------------------
```

Comment:
Why remove SolutionAssemblyInfo? It provides an easy way to change the versioning (and other shared info) for all projects inside the same solution at once. Not sure what the added value is of removing this one.